msys2-runtime/update-patches.sh

```diff
@@ -19,8 +19,11 @@ die "Clean worktree required"
 git rm 0*.patch ||
 die "Could not remove previous patches"
 
+merging_rebase_start="$(git -C src/msys2-runtime \
+    rev-parse --verify --quiet ^HEAD^{/Start.the.merging.rebase})"
+
 git -c core.abbrev=7 -C src/msys2-runtime format-patch -o ../.. --signature=2.9.0 \
-	$base_tag.. ^HEAD^{/Start.the.merging.rebase} ||
+	$base_tag.. $merging_rebase_start ||
```

Comment:
That is missing a `^` before `$merging_rebase_start`...

To make it work, you will probably want to use `${merging_rebase_start:+^$merging_rebase_start}` (which expands to `^<commit>` if that variable is not empty).

Reply:
The `^` is baked into `$merging_rebase_start`

Reply:
Oh, I see, you did it in the `rev-parse` call. I'd rather avoid that magic. As you can see from fooling me: it is very easy to miss if you do it in the `rev-parse` call.

Reply:
Well this way, when it fails the variable is empty. If I added it here you'd get `^` by itself, which I don't think `format-patch` would like.

I guess bash magic could do it correctly if `$merging_rebase_start` is empty. Is that what you want?

Reply:
I think that would change to `$base_tag.. ${merging_rebase_start:+^$merging_rebase_start}`. If you like that better let me know and I'll change it.

Reply:
Yes, that's what I thought. It would have helped me wrap my head around the code quicker, so I would like to ask for that change, indeed.

Reply:
Done!